src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rest/QueryRcaRequestHandler.java

```diff
@@ -48,6 +48,42 @@
  *
  *  <p>To get response for a specific RCA, use:
  *  curl --url "localhost:9650/_opendistro/_performanceanalyzer/rca?name=HighHeapUsageClusterRca" -XGET
+ *
+ *  <p>For temperature profiles, one can get the local node temperature using a request url as:
+ *  curl "localhost:9600/_opendistro/_performanceanalyzer/rca?name=NodeTemperatureRca&local=true"
+ *
+ * <p>The cluster level RCA can only be queried from the elected master using this rest API:
```

Comment:
Given this code goes into AES as well, should we consider moving them to a README for ODFE and AES respectively ?